is08/testConfig.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2018 British Broadcasting Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class TestConfig:
```

Comment:
Whilst I understand the reasoning, the globalConfig construct makes me slightly nervous given (for example) the potential to forget to assign to it in the next test, and have the results become corrupted. I've not got a good solution yet, but I'm wondering if it can in some way be 'reset' between tests in an automatic fashion to reduce this risk?

Reply:
@simonrankine did you have any thoughts on this at all when going back over things?

Reply:
While I see your point I'm going to leave it for now - it's not ideal but the alternatives are even more hairy.